src/OSPSuite.Core/CoreRegister.cs

```diff
@@ -87,15 +97,7 @@ public override void RegisterInContainer(IContainer container)
 
          registerThirdPartyComponents(container);
 
-         //FACTORIES
-         container.RegisterFactory<IDiagramModelFactory>();
```

Comment:
I hvae created aimplementation for all those services.
Not a big deal. StartableProcessFactory is the only one that has parmaeters so I have to dig a bit deeper. It's not used anyways by R